DataAccess/ProductsContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.SqlServer;
+using DataAccess.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Configuration;
+using System.Collections.Specialized;
+
+namespace DataAccess
+{
+    public class ProductsContext : DbContext
+    {
+        public DbSet<Product> Products { get; set; }
+        public DbSet<Category> Categories { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            builder.Entity<Category>()
```

Comment:
Create a separate class for every table configuration. And load it in DbContext.